API/Hardware_API.md

```diff
@@ -184,24 +184,93 @@ low\*     | pin voltage goes low  | Only available via pin.once()
 
 ### PWM pins
 
-**PWM pins** are pulse-width modulated pins ([wiki link](http://en.wikipedia.org/wiki/Pulse-width_modulation)). Essentially, PWM is a digital signal that spends between 0% and 100% of its time pulled high/on (this is its "duty cycle"). You can set the PWM pins to any value between 0 and 1 to approximate an analog signal. PWM is often used to control servo speeds or LED brightness.
+**PWM pins** are pulse-width modulated pins. Essentially, PWM is a digital signal that spends between 0% and 100% of its time pulled high/on (this is its "duty cycle"). You can set the PWM pins to any value between 0 and 1 to approximate an analog signal. PWM is often used to control servo speeds or LED brightness.
```

Comment:
Might want to mention that it works over pulse intervals. One thing I found confusing when initially trying to understand PWM was how it would spend some percentage of its time on. It didn't occur to me that it had specific repeating time intervals over which it was on for a definite period.